Use standard ports for telemetry exposure (#7069)
istio/istio.io: content/en/docs/tasks/observability/gateways/index.md

recommended* for any production or sensitive environment. Insecure access is simpler to set up, but
will not protect any credentials or data transmitted outside of your cluster.

For both options, first follow these steps:

> 1. [Install Istio](/docs/setup/install/istioctl) in your cluster with your desired telemetry addons.

Reviewer: would remove that second clause since the additional telemetry addons are in the next line
```suggestion
1. [Install Istio](/docs/setup/install/istioctl) in your cluster.
```

    * Prometheus: `--set values.prometheus.enabled=true`
    * Tracing: `--set values.tracing.enabled=true`

> 1. Setup the domain to expose addons. In this example, we will expose each addon on a subdomain, such as `grafana.example.com`.

Reviewer: ```suggestion
1. Set up the domain to expose addons. In this example, you expose each addon on a subdomain, such as `grafana.example.com`.
```

    * If you have an existing domain pointing to the external IP address of `istio-ingressgateway`:

    {{< text bash >}}
    $ export INGRESS_DOMAIN=<your.desired.domain>
    {{< /text >}}

    * If you do not have a domain, you may use [`nip.io`](https://nip.io/) which will automatically resolve to the IP address provided. This is not recommended for production usage.

    {{< text bash >}}
    $ export INGRESS_HOST=$(kubectl -n istio-system get service istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
    $ export INGRESS_DOMAIN=${INGRESS_HOST}.nip.io
    {{< /text >}}

### Option 1: Secure access (HTTPS)

A server certificate is required for secure access. Follow these steps to install and configure
server certificates for a domain that you control.

{{< warning >}}
This option covers securing the transport layer *only*. You should also configure the telemetry
addons to require authentication when exposing them externally.
{{< /warning >}}

> In this example we will use self-signed certificates, which may not be appropriate for production usages. For these cases, consider using [cert-manager](/docs/ops/integrations/certmanager/) or other tools to provision certificates. You may also visit the [Securing Gateways with HTTPS](/docs/tasks/traffic-management/ingress/secure-ingress/) task for general information on using HTTPS on the gateway.

Reviewer: ```suggestion
This example uses self-signed certificates, which may not be appropriate for production usages. For these cases, consider using [cert-manager](/docs/ops/integrations/certmanager/) or other tools to provision certificates. You may also visit the [Securing Gateways with HTTPS](/docs/tasks/traffic-management/ingress/secure-ingress/) task for general information on using HTTPS on the gateway.
```

> 1. Setup the certificates. In this example we will use `openssl` to self sign.

Reviewer: ```suggestion
1. Setup the certificates. This example uses `openssl` to self sign.
```

    * Prometheus: `--set values.prometheus.enabled=true`
    * Tracing: `--set values.tracing.enabled=true`

1. Setup the domain to expose addons. In this example, we will expose each addon on a subdomain, such as `grafana.example.com`.

  * If you have an existing domain pointing to the external IP address of `istio-ingressgateway`:

    {{< text bash >}}
    $ export INGRESS_DOMAIN=<your.desired.domain>
    {{< /text >}}

>   * If you do not have a domain, you may use [xip.io](xip.io) which will automatically resolve to the IP address provided

Author: We may want to switch to nip.io actually, seems to have better reviews. Knative is using xip.io and openshift uses nip.io out of the box so it adds some credibility to these. We can probably add a note that you should probably use a real domain for this and this is mostly for demo

Reviewer 2: The insecure access bits were always meant to at least strongly imply demo /quick start usage.  So, adding a little disclaimer about that and the fact that Istio doesn't control nip or xip seems reasonable to me. Thanks for tackling this update!
